pkg/services/ssosettings/models/models.go

```diff
@@ -85,6 +92,20 @@ func (s SSOSettings) ToSSOSettingsDTO() (*SSOSettingsDTO, error) {
 		return nil, err
 	}
 
+	if !includeDefaultSettings {
```

Comment:
We decided to return and store the entire set of settings, so this can be removed! Thanks!